src/Android/Avalonia.Android/Platform/SkiaPlatform/AndroidFramebuffer.cs

```diff
@@ -11,11 +9,11 @@ unsafe class AndroidFramebuffer : ILockedFramebuffer
     {
         private IntPtr _window;
 
-        public AndroidFramebuffer(Surface surface, double scaling)
+        public AndroidFramebuffer(InvalidationAwareSurfaceView surface, double scaling)
         {
             if(surface == null)
                 throw new ArgumentNullException(nameof(surface));
-            _window = ANativeWindow_fromSurface(JNIEnv.Handle, surface.Handle);
+            _window = (surface as IPlatformHandle).Handle;
```

Comment:
`InvalidationAwareSurfaceView.HandleDescriptor` should return `ANativeWindow` instead of `SurfaceView` to make it consistent with the actual handle content. But it is technically a silent breaking change.